helm/starboard-app/templates/priority-level-configuration.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta1
+kind: PriorityLevelConfiguration
+metadata:
+  name: starboard
+spec:
+  type: Limited
+  limited:
+    assuredConcurrencyShares: 5
+    limitResponse:
+      queuing:
```

Comment:
There are other configurable values available here. At the very least, we need to set `handSize`. It can never be greater than the `queueLengthLimit`, which by default is 8, so if someone sets a concurrency limit of 1, this will panic. I suggest setting `handSize = scanJobsconcurrentLimit` to start